File: tests/goldens/rust/valid_minimal.rs
use calculator::*;

// Source: tests/fixtures/valid_minimal.tdd:10
#[test]
fn test_adds_two_numbers() {
    // Input: {'a': 2, 'b': 3}
    let input = vec![("a", 2), ("b", 3)].into_iter().collect::<std::collections::HashMap<_, _>>();
    let result = add(&input);
    assert_eq!(5, result);
}

File: tests/goldens/lua/valid_minimal.lua
local subject = require("calculator")

describe("Calculator", function()
  -- Source: tests/fixtures/valid_minimal.tdd:12
  it("adds two numbers", function()
    local result = subject.add({ a = 2, b = 3 })
    assert.are.same(5, result)
  end)
end)
